apps/web/src/components/job-options-fields.tsx

```
@@ -0,0 +1,231 @@
+import { Button } from '@/components/ui/button'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
+import { Select } from '@/components/ui/select'
+import {
+  type BackoffMode,
+  type JobOptionsFormValue,
+  type JobOptionsValidationErrors,
+  type RetentionMode,
+} from '@/lib/job-options'
```

Biome (2.4.15)

[error] All these imports are only used as types.

(lint/style/useImportType)